sdk/javascript/vectors/catbuffer.js

```diff
@@ -0,0 +1,98 @@
+const nc = require('../src/nem/models');
```

Comment:
i suppose we should move py version from generators/ to vectors/ too to match ?

Reply:
hm?

Reply:
in sdk/python, vectors are in tests/generator 
in sdk/javascript, vectors are in vectors/catbuffer

so i'd move and rename py ones to match new convention